sysvabi64/sysvabi64.rst

```diff
@@ -1592,6 +1592,163 @@ IFUNC requirements for dynamic linkers
 To resolve an ``R_AARCH64_IRELATIVE`` relocation the dynamic linker
 performs the calculation described in AAELF64_ Dynamic Relocations.
 
+Function Multi-versioning
+-------------------------
+
+Function Multi-versioning (FMV) is an Arm C Language Extension that
+lets the compiler generate multiple function versions and auto-dispatch
+between them. Each of the function versions is specialized for a set
+of architecture extensions. The most suitable version is selected
+dynamically. This requires runtime information about the CPU features
+available on the host. FMV is supported on GNU/Linux, Android, and
+many of the BSD operating systems.
+
+On System V platforms Function Multi-versioning is implemented using
+GNU Indirect Functions (IFUNC). The compiler generated IFUNC resolver
+may rely on a runtime library for selecting the most suitable function
```

Comment:
The runtime library does not select the most suitable function, that is done by the ifunc resolver. The runtime library provides the data needed to make that choice.

Perhaps
"
The compiler generated IFUNC resolver may rely on the presence of a global variable ``__aarch64_cpu_features`` provided by the runtime library. It contains information about the available CPU features. The runtime library must also a provide a function ``__init_cpu_features_resolver`` that the IFUNC resolver can call to initialize ``__aarch64_cpu_features``. 
"
I don't think we need the `but it is not a requirement` as we've used `may`.